torch/nn/utils/clip_grad.py

```diff
@@ -50,7 +50,7 @@ def clip_grad_norm_(
         total_norm = norms[0] if len(norms) == 1 else torch.max(torch.stack(norms))
     else:
         norms = []
-        for ((device, _), [grads]) in grouped_grads.items():
+        for ((device, _), ([grads], _)) in grouped_grads.items():  # type: ignore[assignment]
```

Comment:
```suggestion
        for (device, _), ((grads), _) in grouped_grads.items():  # type: ignore[assignment]
```
this would be more accurate, right?

Reply:
The return type is a tuple of a nested list of `Optional[Tensor]` and a list of int. I get that the outer most brackets are redundant, but not the other point.
Isn't `(grads)` a typo of `(grads,)`? `(grads)` would be no different from `grads`.

Reply:
I forgot about the nested list for a moment, this is fine!